### PR TITLE
samples: bluetooth: direct_test_mode: Add support for nRF54L15

### PR DESCRIPTION
**Pull request by maje-emb**
Adapts a DTM sample to the nRF54L15 and nRF54H20.

### CODE REVIEW

File: samples/bluetooth/direct_test_mode/src/dtm.c [hunk 1]
```diff
@@ -1332,7 +1528,11 @@ static void radio_prepare(bool rx)
 	 * between READY event and START task and
 	 * between END event and DISABLE task
 	 */
-#if DIRECTION_FINDING_SUPPORTED
+#if CONFIG_SOC_SERIES_NRF54LX
+	nrf_radio_shorts_set(NRF_RADIO,
+			     NRF_RADIO_SHORT_READY_START_MASK |
+			     NRF_RADIO_SHORT_PHYEND_DISABLE_MASK);
```
Comment (dchat-nordic) [inline samples/bluetooth/direct_test_mode/src/dtm.c:1534]: Maybe this phyend_disable/end_disable could be defined and used here without ifdef for soc?

**Review by dchat-nordic — DISMISSED** [review]
Minor comments and requests.

File: samples/bluetooth/direct_test_mode/boards/nrf54h20dk_nrf54h20_cpurad_soc1.conf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
```
[the remark this answers is not in the document]
Comment (dchat-nordic) [reply samples/bluetooth/direct_test_mode/boards/nrf54h20dk_nrf54h20_cpurad_soc1.conf]: Also in other new files

**Comment by FrancescoSer** [comment]
@dchat-nordic @grochu Please always add a techwriter as reviewer in PRs containing RST files, even if the changes have been approved previously in other repos.

File: samples/bluetooth/direct_test_mode/README.rst [hunk 3]
```diff
@@ -368,6 +368,16 @@ Building and running
...
 
+.. note::
+   On the nRF54H20 PDK, the pins **P2.04** and **P2.05** are assigned to the 2-wire UART interface by default.
+   These pins are also connected to the virtual COM port of the MCU.
+   When testing this example with an external tester connected to the board, it is recommended that unused pins on the board are assigned to the UART interface.
+   You can use the device tree overlay file :file:`nrf54h20dk_uart_test.overlay` with pins **P0.04** and **P0.05** by running the following command:
+
+   .. code-block:: console
+
+      west build samples/bluetooth/direct_test_mode -b nrf54h20dk_nrf54h20_cpurad@soc1 -- -DDTC_OVERLAY_FILE="conf/nrf54h20dk_uart_test.overlay"
```
Comment (FrancescoSer) [inline samples/bluetooth/direct_test_mode/README.rst:379]: @dchat-nordic Is this sample buildable with west from sdk-nrf and its dependencies?

Comment (maje-emb) [reply samples/bluetooth/direct_test_mode/README.rst:379]: Currently, it is not possible to build it using west. Work in progress to add support for these soc's.

**Review by divipillai — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Review by grochu — APPROVED** [review]

**Pull request merged**